store/adapters/rdbms/dal/connection.go

```diff
@@ -220,12 +227,66 @@ func (c *connection) UpdateModel(ctx context.Context, old *dal.Model, new *dal.M
 }
 
 // UpdateModelAttribute alters column on a db table and runs data transformations
-func (c *connection) UpdateModelAttribute(ctx context.Context, sch *dal.Model, old, new *dal.Attribute, trans ...dal.TransformationFunction) error {
-	// not raising not-supported error
-	// because we do not want to break
-	// DAL service model adding procedure
-
-	// @todo implement model column altering
+func (c *connection) UpdateModelAttribute(ctx context.Context, sch *dal.Model,diff *dal.ModelDiff,allowDestructiveChanges bool,hasRecords bool, trans ...dal.TransformationFunction) error {
```

Comment:
Is this fn sign compatible with UpdateModelAttribute on the interface?
Allowing destructive changes is an RDBMS "feature", Do we want DAL to be burdened with it?

Reply:
> Is this fn sign compatible with UpdateModelAttribute on the interface?

It is now because I changed the interface.

This makes sense. I will remove `allowDestructiveChanges` from this signature and its interface and I'll add a private helper in dal/connection.go

Reply:
@darh 
Actually, I just realized that we already put `allowDestructiveChanges` on the dal level when in boot_dal we set the flag on the service. It looks like there's no way currently to set per storage-type flags since dal.New doesn't have any generic way of setting flags.

Should I add a `map[string]bool` type on the dal service so we can have different options per storage type?

Reply:
In the end I kept this in the service and did the appropriate checks there

Reply:
I'm ok if you call `options.DB()` directly from the driver. When we discussed this whole thing the other we said that this is a temporary (🤞 ) solution anyway and when we address it properly (with the user has control via UI) this will all be deprecated.

Just add a line or two of comment when you access this option directly + maybe a note on the option description itself.

Thoguhts?

Reply:
Yeah, that makes it a lot cleaner. Changed to do that.